### ISSUE TITLE
Referenced field is displayed as raw id instead of a link

### ISSUE THREAD

**Issue by xming13** [SoftwareBrothers/adminjs#583]
**Describe the bug**
Referenced field is displayed as raw id instead of a link with the referenced resource's title.

**Installed libraries and their versions**
* "admin-bro": "^3.1.1"
* "@admin-bro/express": "^3.0.0"
* "@admin-bro/sequelize": "^1.1.0"
* "sequelize": "^5.22.3"
* "sequelize-cli": "^5.5.1"

**To Reproduce**
1. Go to list page
2. See the raw id of the referenced field

**Expected behavior**
Referenced field should be displayed as a link with the referenced resource's title

**Screenshots**
![image](https://user-images.githubusercontent.com/6182649/92326582-c2ef4a00-f085-11ea-8d2a-19192cd75953.png)

**AdminBroOptions with schema**
```
const adminBro = new AdminBro({
  rootPath: '/admin',
  loginPath: '/admin/login',
  resources: [
    {
      resource: Item,
      options: {
        parent: sidebarGroups.item,
      },
    },
    {
      resource: User,
      options: {
        parent: sidebarGroups.user,
      },
    }
  ]
});
```

*User*
```
const User = sequelize.define('User', {
    name: DataTypes.STRING,
    email: DataTypes.STRING,
    password: DataTypes.STRING,
    encryptedPassword: DataTypes.STRING,
    role: DataTypes.STRING,
  }, {
    tableName: 'users',
  });
  User.associate = function(models) {
    // associations can be defined here
    User.hasMany(models.Item, { foreignKey: 'ownerId' });
  };
```

*Item*
```
const Item = sequelize.define('Item', {
    // By default, will have the following fields:
    // - id (primary key)
    // - created_at
    // - updated_at

    name: {
      type: DataTypes.STRING,
      field: 'name',
    },
    ownerId: {
      type: DataTypes.BIGINT,
      field: 'owner_id',
      references: {
        model: 'users',
        key: 'id',
      }
    },
  }, {
    tableName: 'items',
    // timestamps: false,
  });

  Item.associate = function(models) {
  };
```

**Comment by wojtek-krysiak** [comment]
1. Item.belongsTo(models.User, { foreignKey: 'owner_id' }) is missing
2. you wrote: User.hasMany(models.Item, { foreignKey: 'ownerId' }); but the field is user_id (not sure, but this might be an issue too)

**Comment by leecheeaun** [comment]
Dear @wojtek-krysiak ,

I managed to reproduce the error using the below Driver/Car model, and can confirm that the raw id of the referenced field is shown instead of a link.

Here is a screenshot:
![image](https://user-images.githubusercontent.com/42826476/92997386-88822300-f545-11ea-812e-8e4f75f1902c.png)

Here is the code: 

AdminBro options with schema
```
const adminBro = new AdminBro({
  rootPath: '/admin',
  loginPath: '/admin/login',
  resources: [
    {
      resource: Driver,
      options: {
        ...DriverResource,
        parent: sidebarGroups.driver,
      },
    },    
    {
      resource: Car,
      options: {
        ...CarResource,
        parent: sidebarGroups.driver,
      },
    },  
  ]
});
```


Model: Car

```
  const Car = sequelize.define('Car', {
    name: {
      type: DataTypes.STRING,
      field: 'name',
    },
    description: {
      type: DataTypes.STRING,
      field: 'description',
    },
    driverId: {
        type: DataTypes.BIGINT,
        field: 'driverId',
        references: {
          model: 'Driver',
          key: 'id',
        }, 
    },
  }, {
    tableName: 'Car',
  });

  Car.associate = function(models) {
    Car.belongsTo(models.Driver, { foreignKey: 'driverId' })
  };

  return Car

};
```

Model: Driver

```
module.exports = (sequelize, DataTypes) => {
  const Driver = sequelize.define('Driver', {   
    name: {
        type: DataTypes.STRING,
        field: 'name'
    },
  }, 
  {
    tableName: 'Driver',
  });
  Driver.associate = function(models) {
    Driver.hasMany(models.Car, { foreignKey: 'carId' });
  };
  return Driver;
};
```

**Comment by wojtek-krysiak** [comment]
i think that you have to change foreignKey: 'carId' to 'driverId' since you dont have carId in the db

**Comment by wojtek-krysiak** [comment]
take a look at this example: https://github.com/SoftwareBrothers/admin-bro-sequelizejs/tree/master/models

**Comment by leecheeaun** [comment]
Dear @wojtek-krysiak , 

Have tried with the example code, but it is still showing the primary key instead of a link to the item's details.

If you would be so kind as to check this link: [https://www.notion.so/leeca89/Referenced-field-is-displayed-as-raw-id-instead-of-a-link-aee9d7c63be44a59bb92bd90c45d079a], I have added screenshots of the front-end UI so the bug is clearer

This has happened only after upgrading AdminBro, Express and Sequelize packages (details are in the link) 

Here is a screenshot of the issue I filed:
![image](https://user-images.githubusercontent.com/42826476/93658131-6cadde00-fa6b-11ea-91d1-cf89ec57ca98.png)

**Comment by wojtek-krysiak** [comment]
ok - I think I know what the problem is. In the meantime, we've updated sequelize version to v6 and apparently in 1.1.0 sequelize v5 doesn't work as it was. I don't want to fix it because we have a task in milestone v3.3 to get rid of populating in the adapters and move everything to admin-bro core package. This will solve the issue. We have delay with v3 (will be released next week - probably Friday so please wait until then.

During then I will notify you If any beta will be released having a fix for this issue.

**Comment by wojtek-krysiak** [comment]
should be fixed in admin-bro@v3.3.0-beta.5 which populates in the core (not in the adapters)

**Comment by wojtek-krysiak** [comment]
btw - there is a new /app folder inside in beta branch: https://github.com/SoftwareBrothers/admin-bro/tree/beta/app - with sequelize setup
